### PR TITLE
V3.8.2 comment

### PR DESCRIPTION
**Pull request by star-e**
Re: #

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

File: cocos/rendering/custom/pipeline.ts [hunk 1]
```diff
@@ -378,21 +385,107 @@ export interface Setter extends RenderNode {
      * @param name @en descriptor name in shader. @zh 填写着色器中的描述符(descriptor)名字
      */
     setSampler (name: string, sampler: Sampler): void;
+    /**
+     * @en Set builtin camera constants of CCCamera, such as cc_matView.
+     * For list of constants, please see CCCamera in cc-global.chunk.
+     * @zh 设置内置相机常量，例如cc_matView。
+     * 具体常量见cc-global.chunk中的CCCamera.
+     * @param camera @en Current camera. @zh 当前相机
```
Comment (minggo) [inline cocos/rendering/custom/pipeline.ts]: `Current camera` seems to be confused. May be can change to `The camera instance to be set` or just ignore it as the function name is meaningful.

File: cocos/rendering/custom/pipeline.ts [hunk 2]
```diff
@@ -378,21 +385,107 @@ export interface Setter extends RenderNode {
...
         light: DirectionalLight,
         csmLevel?: number): void;
+    /**
+     * @en Set builtin spot light frustum and shadow constants.
+     * These constants are used in builtin shadow map.
+     * For list of constants, please see CCShadow in cc-shadow.chunk.
+     * @zh 设置内置探照光视锥与阴影常量。
+     * 这些常量用于内置的阴影。
+     * 具体常量见cc-shadow.chunk中的CCShadow。
+     * @param light @en The directional light. @zh 平行光源
```
Comment (minggo) [inline cocos/rendering/custom/pipeline.ts]: The directional light -> spot light?

File: cocos/rendering/custom/pipeline.ts (hunk 1 above)
Comment (star-e) [reply cocos/rendering/custom/pipeline.ts]: Fixed

File: cocos/rendering/custom/pipeline.ts (hunk 2 above)
Comment (star-e) [reply cocos/rendering/custom/pipeline.ts]: fixed

File: cocos/rendering/custom/pipeline.ts [hunk 3]
```diff
@@ -378,21 +385,107 @@ export interface Setter extends RenderNode {
...
+     * @param light @en The spot light. @zh 探照光源
+     * @param camera @en The camera instance to be set. @zh 当前相机
+     */
     setBuiltinSpotLightConstants (light: SpotLight, camera: Camera): void;
+    /**
+     * @en Set builtin point light and shadow constants.
+     * For list of constants, please check CCShadow in cc-shadow.chunk and CCForwardLight in cc-forward-light.chunk.
+     * @zh 设置内置点光与阴影常量。
+     * 具体常量见cc-shadow.chunk中的CCShadow与cc-forward-light.chunk中的CCForwardLight。
+     * @param light @en The spot light. @zh 点光源
```
Comment (minggo) [inline cocos/rendering/custom/pipeline.ts]: spot light -> point light?

File: cocos/rendering/custom/pipeline.ts [hunk 4]
```diff
@@ -597,6 +721,10 @@ export interface BasicPipeline extends PipelineRuntime {
      * @zh 结束管线构建
      */
     endSetup (): void;
+    /**
+     * @en Enable cpu culling of objects affected by the light. Enabled by default.
+     * @zh 光照计算时，裁切受光源影响的物件。
```
Comment (minggo) [inline cocos/rendering/custom/pipeline.ts]: `Enabled by default` is not mentioned in Chinese.

File: cocos/rendering/custom/pipeline.ts [hunk 5]
```diff
@@ -686,17 +817,65 @@ export interface BasicPipeline extends PipelineRuntime {
         width: number,
         height: number,
         format?: Format): void;
+    /**
+     * @en Add or update buffer.
+     * @zh 添加或更新缓冲
+     * @param name @en Resource name @zh 资源名字
+     * @param size @en Size of the resource @zh 资源的大小
```
Comment (minggo) [inline cocos/rendering/custom/pipeline.ts]: It is better to mention the unit, for example, size in bytes.

File: cocos/rendering/custom/pipeline.ts [hunk 6]
```diff
@@ -686,17 +817,65 @@ export interface BasicPipeline extends PipelineRuntime {
...
         name: string,
         size: number,
         flags: ResourceFlags,
         residency: ResourceResidency): number;
+    /**
+     * @deprecated Method will be removed in 3.9.0
+     * @en Update buffer information.
+     * @zh 更新缓冲的信息
+     * @param name @en Resource name @zh 资源名字
+     * @param size @en Size of the resource @zh 资源的大小
```
Comment (minggo) [inline cocos/rendering/custom/pipeline.ts]: The same as above.

File: cocos/rendering/custom/pipeline.ts (hunk 4 above)
Comment (star-e) [reply cocos/rendering/custom/pipeline.ts]: fixed

File: cocos/rendering/custom/pipeline.ts (hunk 5 above)
Comment (star-e) [reply cocos/rendering/custom/pipeline.ts]: fixed

File: cocos/rendering/custom/pipeline.ts (hunk 6 above)
Comment (star-e) [reply cocos/rendering/custom/pipeline.ts]: fixed

**Review by minggo — APPROVED** [review]

**Pull request merged**